cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -24,6 +24,46 @@ Cask Data Application Platform Release Notes
    :depth: 2
 
 
+`Release 3.0.6 <http://docs.cask.co/cdap/3.0.6/index.html>`__
+=============================================================
+
+Bug Fixes
+---------
+
+- Fixed a problem where a Hadoop filesystem object was being instantiated before the
+  Kerberos keytab login was completed, leading to CDAP processes failing after the initial
+  ticket expired (`CDAP-3660 <https://issues.cask.co/browse/CDAP-3660>`__).
+
+- Fixed a problem where not all dependent services were being stopped when a system service was stopped 
+  (`CDAP-3861 <https://issues.cask.co/browse/CDAP-3861>`__).
+
+- Fixed a problem with dataset performance degradation due to the making of 
+  multiple remote calls for each "get dataset" request 
+  (`CDAP-3890 <https://issues.cask.co/browse/CDAP-3890>`__).
```

Comment:
What is https://issues.cask.co/browse/CDAP-3890 then? It looks identical to me, but maybe I'm missing something?